compiler-rt/test/builtins/Unit/extendhfxf2_test.c

```diff
@@ -13,7 +13,7 @@ long double __extendhfxf2(_Float16 f);
 
 int test_extendhfxf2(_Float16 a, long double expected) {
   long double x = __extendhfxf2(a);
-  __uint16_t *b = (void *)&a;
+  unsigned short *b = (void *)&a;
   int ret = !((isnan(x) && isnan(expected)) || x == expected);
```

Comment:
```suggestion
  int ret = compareResultF80(x, expected);
```
That will do a bitwise comparison and not have issues with NaN.